components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilder.java

```diff
@@ -249,14 +249,14 @@ protected String signJWTWithRSA(JWTClaimsSet jwtClaimsSet, OAuthTokenReqMessageC
             throws IdentityOAuth2Exception {
         try {
             String tenantDomain = request.getOauth2AccessTokenReqDTO().getTenantDomain();
-            int tenantId = request.getTenantID();
+
             if (tenantDomain == null) {
                 tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
             }
```

Comment:
IINM this condition is not needed because we do that when building OAuth2AccessTokenReqDTO